src/Clearinghouse.sol

```diff
@@ -103,6 +109,9 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
         TRSRY = TRSRYv1(getModuleAddress(toKeycode("TRSRY")));
         MINTR = MINTRv1(getModuleAddress(toKeycode("MINTR")));
         ROLES = ROLESv1(getModuleAddress(toKeycode("ROLES")));
+
+        // Approve MINTR for burning OHM (called here so that it is re-approved on updates)
+        ohm.safeApprove(address(MINTR), type(uint256).max);
```

Comment:
If you are using safeApprove here, you may as well use it everywhere. If not, you can just do regular approval here.